packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleTextViewer/SingleTextViewerContainer.js

```diff
@@ -1,121 +1,48 @@
 import PropTypes from 'prop-types'
-import React, { Component } from 'react'
-import request from 'superagent'
+import React from 'react'
+import asyncStates from '@zooniverse/async-states'
 
-import locationValidator from '../../helpers/locationValidator'
 import SingleTextViewer from './SingleTextViewer'
 
-export default class SingleTextViewerContainer extends Component {
-  constructor () {
-    super()
+export default function SingleTextViewerContainer (props) {
+  const {
+    onError,
+    onReady,
+    subject
+  } = props
```

Comment:
Non-blocking comment about code style. React is deprecating `Component.defaultProps` in favour of destructured arguments, for functional components. That's the syntax that's been adopted by Storybook 6+ too, so use the following syntax if you want your arguments to show up in your component's stories.

```suggestion
const defaultSubject = {
  content: '',
  contentLoadingState: asyncStates.initialized
}

export default function SingleTextViewerContainer ({
  onError = () => true,
  onReady = () => true,
  subject = defaultSubject
) {
```